beerocks/bcl/source/beerocks_os_utils.cpp

```diff
@@ -171,7 +171,7 @@ bool os_utils::touch_pid_file(std::string file_path)
 
 int os_utils::redirect_console_std(std::string log_file_name)
 {
-    int fd_log_file_std = open(log_file_name.c_str(), O_CREAT | O_APPEND | O_RDWR, S_IWUSR);
+    int fd_log_file_std = open(log_file_name.c_str(), O_CREAT | O_APPEND | O_RDWR, S_IWUSR | S_IRUSR);
```

Comment:
I had noticed this annoying thing as well :-)

Typically, however, I'd create a log file with `S_IRUSR | S_IWUSR | S_IRGRP | S_IROTH`. Also, I usually use `0644` instead of the symbolic names, because I feel that that is much more readable.

Reply:
Done